spec/ea/area_lookup/test_helper/mock_data_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "spec_helper"
+module EA
+  module AreaLookup
+    module TestHelper
+      describe MockData do
+        let(:mock_data) { described_class.new(:area_lookup) }
+        let(:api_url)   { "http://www.geostore.com/OGC/OGCInterface" }
+
+        describe "#data_for" do
+          context "with :find_by_coordinates" do
+            let(:data) { mock_data.data_for :find_by_coordinates }
+
+            it "mock data should match the keys in the real http response - "\
+               "this catches the scenario where the underlying API has changed but "\
+               "the mock data has not been updated to reflect this" do
+              EA::AreaLookup.config.administrative_area_api_url = api_url
+              coords = EA::AreaLookup::Coordinates.new(easting: 358_205.03, northing: 172_708.07)
+              response = VCR.use_cassette("area_found") do
```

Comment:
Am I right in thinking this line means the project still has a dependency on **VCR**? If so what is the benefit to switching how we mock out calls to external API's? I need to understand because this is something we'll probably want to try and keep consistent across Ruby projects and so far VCR has been what we've used.